src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -141,39 +178,37 @@ export function MessagesList() {
       <FlatList
         ref={flatListRef}
         data={chat.status === ConvoStatus.Ready ? chat.items : undefined}
-        keyExtractor={keyExtractor}
         renderItem={renderItem}
-        contentContainerStyle={{paddingHorizontal: 10}}
-        // In the future, we might want to adjust this value. Not very concerning right now as long as we are only
-        // dealing with text. But whenever we have images or other media and things are taller, we will want to lower
-        // this...probably.
-        initialNumToRender={20}
-        // Same with the max to render per batch. Let's be safe for now though.
-        maxToRenderPerBatch={25}
-        inverted={true}
-        onEndReached={onEndReached}
-        onScrollToIndexFailed={onScrollToEndFailed}
-        onContentSizeChange={onContentSizeChange}
-        onScroll={onScroll}
-        // We don't really need to call this much since there are not any animations that rely on this
-        scrollEventThrottle={100}
+        keyExtractor={keyExtractor}
+        contentContainerStyle={{
+          paddingHorizontal: 10,
+          opacity: hasInitiallyScrolled ? 1 : 0,
+        }}
+        disableVirtualization={true}
```

Comment:
We should revisit if this prop is needed. It _does_ make the scrolling a little bit smoother if we disable virtualization, but it would be good to profile and see how this affects things.